src/data/telemetrics-dirs.conf.in

```diff
@@ -2,6 +2,7 @@ d @localstatedir@/lib/telemetry 0755 telemetry telemetry -
 d @localstatedir@/spool/telemetry 0750 telemetry telemetry -
 d @localstatedir@/log/telemetry 0750 telemetry telemetry -
 d @localstatedir@/log/telemetry/records 0750 telemetry telemetry -
+d @localstatedir@/tmp/telemetry/ 0740 telemetry telemetry -
```

Comment:
0740? Should this be 0750?

Reply:
I was trying to apply the most narrow permission. I don't feel strong about it. Will change to 750.